packages/docs/concepts/connections-and-requests.yaml

```diff
@@ -177,6 +177,273 @@ _ref:
+                    properties:
+                      url: /users
+            ```
+
+            This request should be called when the page is mounted. To do this, the `Request` action will be used during the `onMount` event of the home page.
+
+            ###### lowdefy.yaml
+            ```yaml
+            lowdefy: LOWDEFY_VERSION
+
+            connections:
+              - id: my_api
+                type: AxiosHttp
+                properties:
+                  baseURL: https://jsonplaceholder.typicode.com
+
+            pages:
+              - id: home
+                type: PageHeaderMenu
+
+                requests:
+                  - id: get_users
+                    type: AxiosHttp
+                    connectionId: my_api
+                    properties:
+                      url: /users
+
+                events:
+                  onMount:
+                    - id: fetch_get_users
+                      type: Request
+                      params: get_users
+            ```
+
+            A block can be added to the page, that can use or display the data returned by this request. The `_request` operator is used to get the the results of the request.
+            In this example, a `Selector` block will be added. The `_array.map` operator is used to get the data in a form that can be used by the Selector.
+
+            ###### lowdefy.yaml
+            ```yaml
+            lowdefy: LOWDEFY_VERSION
+
+            connections:
+              - id: my_api
+                type: AxiosHttp
+                properties:
+                  baseURL: https://jsonplaceholder.typicode.com
+
+            pages:
+              - id: home
+                type: PageHeaderMenu
+
+                requests:
+                  - id: get_users
+                    type: AxiosHttp
+                    connectionId: my_api
+                    properties:
+                      url: /users
+
+                events:
+                  onMount:
+                    - id: fetch_get_users
+                      type: Request
+                      params: get_users
+
+                blocks:
+                  - id: user_id
+                    type: Selector
+                    properties:
+                      title: Select a User
+                      options:
+                        _array.map:
+                          - _request: get_users.data
+                          - _function:
+                              label:
+                                __args: 0.name
+                              value:
+                                __args: 0.id
+            ```
+
+            To get the posts made by the selected user, a new request should be created and called once the value of the Selector changes.
+            To do this, create a new request that will make us of a `payload` to get the posts with the selected user id. Add an `onChange` event on the `Selector` block, during which the new request is called.
```

Comment:
*use

Reply:
Change made in d611ea5901b9c9e552b83c41f6a7f5b674515f34